### ISSUE TITLE
Stopword support

### ISSUE THREAD

**Issue by Ovid** [typesense/typesense#248]
## Description

It would be great if we could configure stopwords, even if it's only at the server level rather than per API key.

For example, if I own a site rating hotels, I want "hotel" to be a stop word. If I search for a beach hotel called "The Rusty Pelican" (no hotel in name) and I search for "Pelican Hotel", I don't want "Hyatt-Rengency Hotel" showing up just because "hotel" is in the name.

## Steps to reproduce

Not applicable.

## Expected Behavior

Not applicable.

## Actual Behavior

Not applicable.

## Metadata

Not available as of 0.19.0.

**Comment by A4TIF** [comment]
Hi, any progress on this? It is crucial for our business to have some politically banned words to not be showing in the search results. Thanks and keep up the good work with Typesense.

**Comment by jasonbosco** [comment]
Stop words support is now available in `0.26.0.rc42`. 

Here's how to use it:

Create a stop words set:

```bash
curl "http://localhost:8108/stopwords/stopword_set1" \ 
  -X PUT \
  -H "Content-Type: application/json" \
  -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}" \
  -d '{"stopwords": ["the", "a", "an"], "locale": "en"}'
```

To use this stop word set in a search query, use the `stopwords` search parameter and specify the stop words set name from above:

```bash
curl -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}" \
  "http://localhost:8108/collections/collection_name/documents/search?q=the+brand&query_by=title&stopwords=stopword_set1"
```

Other API operations:

Update stop words set:

```bash
curl "http://localhost:8108/stopwords/stopword_set1" \
  -X PUT \
  -H "Content-Type: application/json" \
  -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}" \
  -d '{"stopwords": ["Germany", "France", "Italy"], "locale": "en"}'
```

Delete stop words set:

```bash
curl "http://localhost:8108/stopwords/stopword_set1" \
  -X DELETE \
  -H "Content-Type: application/json" \
  -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}"
```

**Comment by jasonbosco** [comment]
Released in v26
